inbox/models/thread.py

```diff
@@ -117,9 +117,11 @@ def most_recent_sent_date(self):
         sorted_messages = sorted(
             self.messages, key=lambda m: m.received_date, reverse=True
         )
-        for m in sorted_messages:
-            if "sent" in [c.name for c in m.categories] or (m.is_draft and m.is_sent):
-                sent_recent_date = m.received_date
+        for message in sorted_messages:
+            if "sent" in [
+                category.name for category in message.categories if category
```

Comment:
`if category` is the part that I added to filter out `None` categories that come from missing categories because of missing FK.